samples/Azure.Management.Storage/Generated/BlobContainerCollection.cs

```diff
@@ -32,8 +33,9 @@ protected BlobContainerCollection()
         }
 
         /// <summary> Initializes a new instance of the <see cref="BlobContainerCollection"/> class. </summary>
-        /// <param name="parent"> The resource representing the parent resource. </param>
-        internal BlobContainerCollection(ArmResource parent) : base(parent)
+        /// <param name="armClient"> The client parameters to use in these operations. </param>
+        /// <param name="id"> The identifier of the parent resource that is the target of operations. </param>
+        internal BlobContainerCollection(ArmClient armClient, ResourceIdentifier id) : base(armClient, id)
```

Comment:
Curious: is this the resource client collection pattern we discussed?

Reply:
yes this is a [Resource]Collection client which implements the IEnumerable, Exists, GetAll, etc